### PR TITLE
feat(PS2): service profile summary

### PR DESCRIPTION
**Pull request by strrife**

### CODE REVIEW

File: packages/ps2/src/util/i18next.ts [hunk 1]
```diff
@@ -2,11 +2,24 @@ import i18n from 'i18next';
...
 import Backend from 'i18next-http-backend';
+import Countries from 'i18n-iso-countries';
+import CountriesEn from 'i18n-iso-countries/langs/en.json';
+import enUS from 'date-fns/locale/en-US';
+
+Countries.registerLocale(CountriesEn);
+
+// mishka vodka balalayka
+if (CountriesEn.countries.RU)
+  CountriesEn.countries.RU = 'Mother Russia' as unknown as string[];
```
Comment (cwagner22) [inline packages/ps2/src/util/i18next.ts:13]: Hmm ok xD

File: packages/ps2/src/views/TraderService/components/ServiceProfileContainer/atoms/ServiceGrowthChart.tsx [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      {/*    </ZigButton>*/}
+      {/*  ))}*/}
+      {/*</ButtonGroup>*/}
+      {/*<ZigSelect*/}
+      {/*  label={''}*/}
+      {/*  value={chartType}*/}
+      {/*  onChange={(v) => setChartType(v)}*/}
+      {/*  options={chartTypeOptions}*/}
+      {/*></ZigSelect>*/}
+      {/*</Box>*/}
```
Comment (cwagner22) [inline packages/ps2/src/views/TraderService/components/ServiceProfileContainer/atoms/ServiceGrowthChart.tsx:53]: Left on purpose or can be removed?

File: packages/ps2/src/views/TraderService/components/ServiceProfileContainer/atoms/ServiceManagerDescription.tsx [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+            {service.ownerCountry &&
+              flags[service.ownerCountry?.toLocaleUpperCase()] && (
+                <Tooltip
+                  title={t('owner-from', {
+                    country: country || service.ownerCountry,
+                  })}
+                >
+                  <CountryFlag
+                    dangerouslySetInnerHTML={{
+                      __html: flags[service.ownerCountry?.toLocaleUpperCase()],
```
Comment (cwagner22) [inline packages/ps2/src/views/TraderService/components/ServiceProfileContainer/atoms/ServiceManagerDescription.tsx:58]: why do we need to set as html?

**Review by cwagner22 — APPROVED** [review]

File: packages/zignaly-ui/src/components/inputs/ZigButton/types.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import { MenuPlacement, Props as SelectProps } from "react-select";
+
+export type ZigSelectOption<T> = {
+  value: T;
+  label: string | JSX.Element;
+  disabled?: boolean;
+};
+
+export type ZigSelectProps<T> = Omit<
```
[the remark this answers is not in the document]
Comment (strrife) [reply packages/zignaly-ui/src/components/inputs/ZigButton/types.ts:9]: good catch!

File: packages/ps2/src/views/TraderService/components/ServiceProfileContainer/atoms/ServiceGrowthChart.tsx (hunk 2 above)
Comment (strrife) [reply packages/ps2/src/views/TraderService/components/ServiceProfileContainer/atoms/ServiceGrowthChart.tsx:53]: good catch, removed

**Pull request merged**